### DISCUSSION TITLE
package:find_package may overlook the header files

### DISCUSSION THREAD

**Discussion by xq114** [xmake-io/xmake#4337] [General]
https://github.com/xmake-io/xmake/blob/master/xmake/core/package/package.lua#L1723-L1746

这里是否漏传了includes参数？在 https://github.com/xmake-io/xmake-repo/blob/dev/packages/l/libiconv/xmake.lua#L27 等一些地方有检测头文件，但实际执行的时候没有检测头文件

**Comment by waruqi** [comment]
这里为啥会传 includes 。。我从来没支持过这个参数。

**Reply by xq114** [reply]
https://github.com/xmake-io/xmake/blob/master/xmake/modules/package/manager/system/find_package.lua#L43-L62 但是find_package里是有includes的查找的

**Reply by waruqi** [reply]
哦，这里啊，我都忘记了。

现在传了，再试下
